dotnet/Firely.Server.MessageSender/UserInputProcessor.cs

```diff
@@ -62,6 +65,24 @@ public async Task ProcessUserInput()
                     }
                     badArguments = true;
                     break;
+                case "dir":
+                    if (!arguments.Any())
+                    {
+                        if (string.IsNullOrEmpty(_importOptions.ImportDirectory))
```

Comment:
see other comment. Mixing interactive param with config setting is confusing for me.